ci-operator/step-registry/openshift-qe/installer/bm/day2/label/openshift-qe-installer-bm-day2-label-commands.sh

```
@@ -123,3 +123,12 @@ if [ ${TELCO} == "true" ]; then
     oc get nodes -l node-role.kubernetes.io/worker --show-labels
   fi
 fi
+
+if [ "${NODE_DENSITY_CNI_WORKLOAD}" == "true" ]; then
+  # Label the nodes with node-density-cni-workload
+  for node in `oc get nodes -l node-role.kubernetes.io/worker= --no-headers | head -${LABEL_NUM_NODES} | awk '{print $1}'`; do
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)